openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/5-infra-chaos/ECBZ-kubelet-failure-cstor/kubelet-failure-cstor

```diff
@@ -0,0 +1,247 @@
+
+if [ "$?" != "0" ]; then
+python3 openebs-konvoy-e2e/utils/result/result_update.py $job_id ECBZ 5-infra-chaos "stop the kubelet service on the node where application is running and check the behaviour" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+################
+# LitmusBook 2 #
+################
+
+run_id="kubelet-failure-cstor";test_name=$(bash openebs-konvoy-e2e/utils/generate_test_name testcase=busybox-liveness metadata=${run_id})
+echo $test_name
+
+cd litmus
+cp apps/busybox/liveness/run_litmus_test.yml busybox_liveness_kubelet_failure_cstor.yml
+
+# Update the environmental variables in litmus job spec.
+
+: << EOF
+  ----------------------------------------------------------------------------------------------------------------------
+ | specAttribute     | kind   |         default value               | test specifc value                               |
+  ---------------------------------------------------------------------------------------------------------------------|
+ | appNamespace      | env    | app-busybox-ns                      | kubelet-failure-cstor                                     | 
+ | Litmus job label  | label  | liveness: litmus-busybox-liveness   | liveness: busybox-liveness-kubelet-failure-cstor           |
+  ----------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/value: app-busybox-ns/value: kubelet-failure-cstor/g' \
+-e 's/liveness: litmus-busybox-liveness/liveness: busybox-liveness-kubelet-failure-cstor/g' busybox_liveness_kubelet_failure_cstor.yml
+
+sed -i '/command:/i \
+          - name: RUN_ID\
+            value: '"$run_id"'\
+' busybox_liveness_kubelet_failure_cstor.yml
+
+cat busybox_liveness_kubelet_failure_cstor.yml
+
+# Run the Litmus job and get the details of the litmus job from litmus_job_runner utils.
+bash ../openebs-konvoy-e2e/utils/litmus_job_runner label='liveness:busybox-liveness-kubelet-failure-cstor' job=busybox_liveness_kubelet_failure_cstor.yml
+cd ..
+# Get the cluster state Once the litmus jobs completed.
+bash openebs-konvoy-e2e/utils/dump_cluster_state;
+# Update the e2e event for the job.
+bash openebs-konvoy-e2e/utils/event_updater jobname:kubelet-failure $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 openebs-konvoy-e2e/utils/result/result_update.py $job_id ECBZ 5-infra-chaos "stop the kubelet service on the node where application is running and check the behaviour" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+################
+# LitmusBook 3 #
+################
+
+run_id="cstor";test_name=$(bash openebs-konvoy-e2e/utils/generate_test_name testcase=svc-failure-chaos metadata=${run_id})
+echo $test_name
+
+cd litmus
+cp experiments/chaos/kubernetes/service_failure/run_litmus_test run_kubelet_failure_test_cstor.yml
```

Comment:
run_litmus_test.yml instead of run_litmus_test

Reply:
Modified ! @somesh2905